util/types/json/impl.go

```diff
@@ -41,7 +41,14 @@ func normalize(in interface{}) JSON {
 		return jsonLiteralFalse
 	case nil:
 		return jsonLiteralNil
+	case int64:
+		return jsonInt64(t)
 	case float64:
+		// json.Unmarshal converts all number to float64,
+		// so we need convert integer back.
+		if float64(int64(t)) == t {
```

Comment:
This maybe a better solution: https://stackoverflow.com/questions/22343083/json-marshaling-with-long-numbers-in-golang-gives-floating-point-number